packages/browser/src/Session.ts

```diff
@@ -231,7 +231,7 @@ export class Session extends EventEmitter {
    * @param url The URL from which data should be fetched.
    * @param init Optional parameters customizing the request, by specifying an HTTP method, headers, a body, etc. Follows the [WHATWG Fetch Standard](https://fetch.spec.whatwg.org/).
    */
-  fetch = async (url: RequestInfo, init?: RequestInit): Promise<Response> => {
+  fetch: typeof fetch = async (url, init) => {
```

Comment:
This should prevent the issues we've seen in the future, as it reuses the typing from lib.dom.ts directly rather than declaring a compatible type.